### PR TITLE
ci: bump otp version to 23.3.4.18-1

### PR DESCRIPTION
**Pull request by id**
- [ ] Added tests for the changes
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/` dir
- [ ] For EMQX 4.x: `appup` files updated (execute `scripts/update-appup.sh emqx`)
- [x] For internal contributor: there is a jira ticket to track this change
- [ ] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

### CODE REVIEW

File: .github/workflows/run_automate_tests.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: erlef/setup-beam@v1
       with:
-        otp-version: "23.3.4.13"
+        otp-version: "23.3.4.17"
```
Comment (thalesmg) [inline .github/workflows/run_automate_tests.yaml:32]:
```suggestion
        otp-version: "23.3.4.18"
```
?

File: .github/workflows/run_fvt_tests.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ jobs:
         - uses: actions/checkout@v1
         - uses: erlef/setup-beam@v1
           with:
-            otp-version: "23.3.4.9"
+            otp-version: "23.3.4.17"
```
Comment (thalesmg) [inline .github/workflows/run_fvt_tests.yaml:18]: Here and below: shouldn't it be 18?

**Review by thalesmg — APPROVED** [review]

File: .github/workflows/run_fvt_tests.yaml (hunk 2 above)
Comment (id) [reply .github/workflows/run_fvt_tests.yaml:18]: Wasn't sure if 23.3.4.18 was available for this action when I was fixing it, but yeah, most likely it should be fine to put 18 here as well.
For the sake of the original issue - 23.3.4.15 is enough to address [CVE-2022-37026](https://cve.report/CVE-2022-37026).

Comment (lafirest) [reply .github/workflows/run_fvt_tests.yaml:18]: When I update the emqx-builder repo, there only was 23.3.4.17.
If we want to use the newest version of OTP, we need update the emqx-builder again, do we need ?

Comment (id) [reply .github/workflows/run_fvt_tests.yaml:18]: 17 should be ok I thinj

**Review by lafirest — APPROVED** [review]

**Comment by zmstone** [comment]
picked to this pr: https://github.com/emqx/emqx/pull/9425

**Pull request closed without merging**